example/example-metadata.ttl

```diff
@@ -22,308 +22,356 @@
-<dietrich> dsp-repo:hasSpatialCoverage "Switzerland".
-<dietrich> dsp-repo:hasFunder "Schweizerischer Nationalfonds SNF" .
-<dietrich> dsp-repo:hasFunder "Kanton Aargau (Lotteriefonds)" .
-<dietrich> dsp-repo:hasFunder "Kanton Schwyz (Lotteriefonds)" .
-<dietrich> dsp-repo:hasFunder "Kanton Thurgau (Kulturamt)" .
-<dietrich> dsp-repo:hasFunder "AVINA-Stiftung" .
-<dietrich> dsp-repo:hasFunder "Ernst Göhner-Stiftung" .
-<dietrich> dsp-repo:hasFunder "Cogito Stiftung" .
-<dietrich> dsp-repo:hasURL "http://www.dietrich-edition.unibe.ch/" .
-<dietrich> dsp-repo:hasShortcode "0000" .
-<dietrich> dsp-repo:hasAlternateName "Dietrich-Projekt" .
-<dietrich> dsp-repo:hasPublication "yes" .
-
-<dietrich-dataset> rdf:type dsp-repo:Dataset .
-<dietrich-dataset> dsp-repo:hasIdentifier "fakeARKstring".
-<dietrich-dataset> dsp-repo:hasEncodingFormat "image" .
-<dietrich-dataset> dsp-repo:hasEncodingFormat "text" .
-<dietrich-dataset> dsp-repo:hasDocumentation "yes" .
-<dietrich-dataset> dsp-repo:hasLicense "fakelicense" .
-<dietrich-dataset> dsp-repo:hasConditionsOfAccess "Public Access" .
-<dietrich-dataset> dsp-repo:hasUsageInfo "fakecitation".
-<dietrich-dataset> dsp-repo:hasStatus "data will be added continuously" .
-<dietrich-dataset> dsp-repo:hasDatePublished "2019" .
-<dietrich-dataset> dsp-repo:hasLanguage "DE" .
-<dietrich-dataset> dsp-repo:isPartOf <dietrich> .
-
-<dietrich-dataset> dsp-repo:hasQualifiedAttribution [
-    a prov:Attribution;
-    prov:agent <dietrich-schwarz>;
-    dsp-repo:hasRole "editor";
+    prov:agent <test-berry>;
+    dsp-repo:hasRole "contributor";
 ] .
-
-<dietrich-dataset> dsp-repo:hasQualifiedAttribution [
+<test-dataset> dsp-repo:hasQualifiedAttribution [
     a prov:Attribution;
-    prov:agent <dietrich-rohr>;
+    prov:agent <test-coleman>;
     dsp-repo:hasRole "editor";
 ] .
 
-<dietrich-schwarz> rdf:type dsp-repo:Person .        
-<dietrich-schwarz> dsp-repo:hasGivenName "G." .        
-<dietrich-schwarz> dsp-repo:hasFamilyName "Schwarz-Zanetti" .
-<dietrich-schwarz> dsp-repo:isMemberOf <dietrich-organization> .
-<dietrich-schwarz> dsp-repo:hasJobTitle "Dr." .
-
-<dietrich-rohr> rdf:type dsp-repo:Person .        
-<dietrich-rohr> dsp-repo:hasGivenName "Christian" .        
-<dietrich-rohr> dsp-repo:hasFamilyName "Rohr" .
-<dietrich-rohr> dsp-repo:isMemberOf <dietrich-organization> .
-<dietrich-rohr> dsp-repo:hasJobTitle "Prof. Dr." .
-
-<dietrich-organization> rdf:type dsp-repo:Organization . 
-<dietrich-organization> dsp-repo:hasOrganizationName "Universität Bern" .
\ No newline at end of file
+<test-abraham> rdf:type dsp-repo:Person . 
+<test-abraham> dsp-repo:hasGivenName "Stewart" . 
+<test-abraham> dsp-repo:hasFamilyName "Abraham" . 
+<test-abraham> dsp-repo:hasEmail "stewart.abraham@test.ch" . 
+<test-abraham> dsp-repo:hasAddress "Teststrasse 12, 4000 Basel" .
+<test-abraham> dsp-repo:isMemberOf <test-dasch> .
+<test-abraham> dsp-repo:hasJobTitle "Dr." .
+<test-abraham> dsp-repo:hasIdentifier [
```

Comment:
As far as I know, this should be enough (if we don't create a class for `Identifier`) 
```ttl
<test-abraham> dsp-repo:hasIdentifier "https://orcid.org/0000-0002-1825-0097"
```

Reply:
Unfortunately, it is not, please see: https://schema.org/identifier

Reply:
Maybe we're turning in circles here... but two options I figured out just now:

1) if we only want the URL, this also does the trick:

```ttl
dsp-repo:PersonShape
    a sh:NodeShape ;
    sh:targetClass dsp-repo:Person ;
     sh:property [ 
        sh:path dsp-repo:hasIdentifier ;
        sh:name "Identifier" ;
        sh:description "Identifier of the person"@en ;
        sh:property schema:url ;
     ] ;
```

this works with:

```ttl
<test-abraham> rdf:type dsp-repo:Person . 
<test-abraham> dsp-repo:hasIdentifier "https://orcid.org/0000-0002-1825-0097" .
```

2) the following works for a "qualified ID":
```ttl
dsp-repo:PersonShape
    a sh:NodeShape ;
    sh:targetClass dsp-repo:Person ;
     sh:property [ 
        sh:path dsp-repo:hasIdentifier ;
        sh:name "Identifier" ;
        sh:description "Identifier of the person"@en ;
        sh:property schema:url ;
        sh:property [
            sh:path dsp-repo:hasIdentifierScheme ;
            sh:datatype xsd:string ;
            sh:minCount 1 ;
        ]
     ] .
```

allows for a person with:

```ttl
<test-abraham> rdf:type dsp-repo:Person . 
<test-abraham> dsp-repo:hasIdentifier <test-abraham-id> .
<test-abraham-id> schema:url "https://orcid.org/0000-0002-1825-0097" .
<test-abraham-id> dsp-repo:hasIdentifierScheme "Blah" .
```

... is that of any help at all?

Reply:
Thx for your input! I have the simplest solution at all, all identifiers are deleted ;-)